.eslintrc.json

```diff
@@ -5,6 +5,7 @@
     "prettier"
   ],
   "rules": {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
Do we really need this? or should we ignore this rule in specific places? 💭

Reply:
Removed rules section and place into the specific file https://github.com/Lahphim/react-surveyor-challenge/pull/65/commits/3d5c2b4f3bca11fc59665ef03c10d109bfd47519